src/hooks/VoicePlayer/dux/actionTypes.ts

```diff
@@ -0,0 +1,11 @@
+export type InitializeAudioUnit = 'INITIALIZE_AUDIO_UNIT';
+export type SetCurrentPlayer = 'SET_CURRENT_PLAYER';
+export type OnVoicePlayerPlay = 'ON_VOICE_PLAYER_PLAY';
+export type OnVoicePlayerPause = 'ON_VOICE_PLAYER_PAUSE';
+export type OnCurrentTimeUpdate = 'ON_CURRENT_TIME_UPDATE';
```

Comment:
I added types, but I don't know it's a good idea

Reply:
whynot use the uppercase name as the const type? example -> `INITIALIZE_AUDIO_UNIT`

Reply:
This is fine for now~ dont need to change

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/398/files#diff-c23f1837a7a5060b1a091e4fa42ddd54f782ad6f9f563e79220549107ad7357c > use this pattern, its cleaner ~